http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -40,15 +43,14 @@
     public static final String JAVAX_NET_SSL_TRUST_STORE = "javax.net.ssl.trustStore";
     public static final String JAVAX_NET_SSL_TRUST_STORE_PASSWORD = "javax.net.ssl.trustStorePassword";
 
-    private boolean http2Enabled;
-    private Duration connectTimeout = new Duration(1, TimeUnit.SECONDS);
+    private Duration connectTimeout = new Duration(1, SECONDS);
```

Comment:
These unrelated changes make it hard to read, can you pull this into a separate commit?